tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+# testing...
+
+
+import os
+import pytest
+
+# Dirty hack for adding the module discoverable (for now)
+import sys
+
+cwd = os.getcwd()
+sys.path.append(
+    cwd + "/mender-image-tests/tests"
+)
```

Comment:
Alternatively, something like this should also work:
```suggestion
sys.path.append(
    os.path.join(os.path.dirname(__file__), "mender-image-tests/tests")
)
```

Reply:
Nice, this is better